exporter/opentelemetry-exporter-otlp-proto-grpc/tests/test_otlp_exporter_mixin.py

```diff
@@ -26,20 +26,213 @@
 from google.rpc.error_details_pb2 import (  # pylint: disable=no-name-in-module
     RetryInfo,
 )
-from grpc import Compression
+from grpc import Compression, server
```

Comment:
nit: We're importing `StatusCode` from `grpc` via `opentelemetry.exporter.otlp.proto.grpc.exporter` instead of directly.
```suggestion
from grpc import Compression, server, StatusCode
```

Reply:
Nice, done